api/goldens/aptos_api__tests__transactions_test__test_get_transactions_output_genesis_transaction.json

```diff
@@ -11,14 +11,24 @@
     "vm_status": "Executed successfully",
     "accumulator_root_hash": "",
 =======
+<<<<<<< HEAD
```

Comment:
@bchocho, n00b question: this looks a merge conflict, right? How did we commit this? 😄

Reply:
Wow, that's pretty bad. Let me try to fix this. Surprised that it this would land.

Reply:
I think it wasn't caught because the test is marked as ignore. So it shouldn't block anyone ATM. But yes, let me fix this.